certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/HTMLTempalteZip.java

```diff
@@ -20,7 +23,7 @@
     /**
      * html zip file url
      */
-    private URL zipUrl;
+    private static URL zipUrl;
```

Comment:
how this could be static? Are we saying there is only HTML template that this service could work with? This defeats the purpose of a service. Please remove all static keywords.